src/PaymentGateways/PayPalCommerce/ScriptLoader.php

```diff
@@ -142,12 +142,12 @@ public function loadPublicAssets()
 
         /**
          * List of PayPal query parameters: https://developer.paypal.com/docs/checkout/reference/customize-sdk/#query-parameters
+         * @unreleased Removes locale from parameters.
```

Comment:
Can you make this past tense, so it reads as a change log?

Reply:
Resolved: https://github.com/impress-org/givewp/pull/6791/commits/0135ebd2b23fb5e18277467cc21c9e789bb22ca7